### DISCUSSION TITLE
Welcome to tidybiology Discussions!

### DISCUSSION THREAD

**Discussion by matthewhirschey** [tidybiology/tidybiology#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BAREJAA** [comment]
Hi! My name is Akshay, and I help develop Tidybiology content and teach classes. I look forward to meeting fellow R/tidyverse/data science fans!
